pkg/hope/hypervisors/esxi_hypervisor_test.go

```diff
@@ -0,0 +1,105 @@
+package hypervisors
```

Comment:
Could (should?) be merged into master before the rest of these changes. 

Would just need to be adjusted to create a packerJson, and then this diff becomes a relatively simple update.